### PR TITLE
Check ignoreID's in TF publishing

### PR DESCRIPTION
**Pull request by jack-digilabs**
Make sure aruco_detect checks ignoreID's even when publishing TF's.

### CODE REVIEW

File: aruco_detect/src/aruco_detect.cpp [hunk 1]
```diff
@@ -372,7 +376,7 @@ void FiducialsNode::imageCallback(const sensor_msgs::ImageConstPtr & msg) {
                          rvecs[i][0], rvecs[i][1], rvecs[i][2]);
 
                 if (std::count(ignoreIds.begin(), ignoreIds.end(), ids[i]) != 0) {
-                    ROS_INFO("Ignoring id %d", ids[i]);
```
Comment (jim-v) [inline aruco_detect/src/aruco_detect.cpp]: Could we go back to `ROS_INFO`?

File: aruco_detect/src/aruco_detect.cpp [hunk 2]
```diff
@@ -329,6 +329,10 @@ void FiducialsNode::imageCallback(const sensor_msgs::ImageConstPtr & msg) {
         ROS_INFO("Detected %d markers", (int)ids.size());
 
         for (size_t i=0; i<ids.size(); i++) {
+	    if (std::count(ignoreIds.begin(), ignoreIds.end(), ids[i]) != 0) {
+	        ROS_WARN("Ignoring id %d", ids[i]);
```
Comment (jim-v) [inline aruco_detect/src/aruco_detect.cpp]: Can we please have this as `ROS_INFO`

**Review by jim-v — CHANGES_REQUESTED** [review]

File: aruco_detect/src/aruco_detect.cpp (hunk 2 above)
Comment (jack-digilabs) [reply aruco_detect/src/aruco_detect.cpp]: Sure thing!

**Review by jack-digilabs — COMMENTED** [review]
Requested changes made

File: aruco_detect/src/aruco_detect.cpp (hunk 1 above)
Comment (jack-digilabs) [reply aruco_detect/src/aruco_detect.cpp]: Done!

File: aruco_detect/src/aruco_detect.cpp [hunk 3]
```diff
@@ -329,6 +329,10 @@ void FiducialsNode::imageCallback(const sensor_msgs::ImageConstPtr & msg) {
         ROS_INFO("Detected %d markers", (int)ids.size());
 
         for (size_t i=0; i<ids.size(); i++) {
+	    if (std::count(ignoreIds.begin(), ignoreIds.end(), ids[i]) != 0) {
```
Comment (rohbotics) [inline aruco_detect/src/aruco_detect.cpp:332]: I believe `std::find` is the canonical way to check for existence.

Consider using: `std::find(ignoreIds.begin(), ignoreIds.end(), ids[i]) != ignoreIds.end()`

Comment (jack-digilabs) [reply aruco_detect/src/aruco_detect.cpp:332]: That code is copied from later on. I'm OK with whichever method you prefer!

**Review by jim-v — APPROVED** [review]

File: aruco_detect/src/aruco_detect.cpp (hunk 3 above)
Comment (jim-v) [reply aruco_detect/src/aruco_detect.cpp:332]: I think @rohbotics and I had some discussion about that at the time ignoring was initially implemented ignoring and settled on `std::count`.  Either is fine with me.

**Comment by rohbotics** [comment]
Thanks for your contribution @jack-digilabs

**Pull request merged**